.github/actions/publish/action.yml

```diff
@@ -0,0 +1,14 @@
+name: Publish Package
+description: 'Publish the package to Cocoapods'
+inputs:
+  dry_run:
+    description: 'Is this a dry run. If so no package will be published.'
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Push to cocoapods
+      if: ${{ inputs.dry_run == 'false' }}
+      shell: bash
+      run: pod trunk push LaunchDarkly.podspec --allow-warnings --verbose
```

Comment:
Guessing the tool itself doesn't have a dry run mode?

Reply:
Correct. I did check originally but it doesn't appear to support it. We do perform a pod spec lint every time and that pretty much does everything except push so we should be _somewhat_ shielded.